src/test/java/com/googleinterns/zoomtube/servlets/DiscussionServletTest.java

```diff
@@ -279,7 +279,7 @@ private Entity createTestCommentEntity(int lectureId) {
 
   private List<Comment> getCommentsFromJson(String json) {
     Gson gson = new GsonBuilder().registerTypeAdapterFactory(GenerateTypeAdapter.FACTORY).create();
-    Type listType = new TypeToken<ArrayList<Comment>>() {}.getType();
+    Type listType = (new ArrayList<Comment>()).getClass();
```

Comment:
I remember seeing this change in a different pr

Reply:
oop, i'll merge with master